### ISSUE TITLE
Option to allow pausing by clicking on the video itself 

### ISSUE THREAD

**Issue by Zcc09** [Stremio/stremio-features#406]
**Is your feature request related to a problem? Please describe.**
It's often that I try to pause a video I'm watching by clicking on the video itself not the play/pause button and it feels out of place since many popular media players like MPC and even stremio web allows for that to happen.

**Describe the solution you'd like**
An option in the settings that would allow for pausing the video by pressing on the video itself since I understand some people don't want that feature so I toggle would be perfect.

**Describe alternatives you've considered**
I tried using external media players as well as the web app but that feels like a work around and always takes that one extra step.

**Comment by WiretapStudios** [comment]
Agree. Every other player I use on desktop has this (VLC, YouTube, Max.com). It's hard remembering to click the tiny little play button to pause. I'm trying to train myself to use the spacebar, but I'm often in another window and by clicking to the app, I could just already be clicking the video to pause.

**Comment by Zcc09** [comment]
> You can press space to pause the video

This is not what this issue is about. 
This is about adding a feature that other apps and stremio itself has already in it's web version, it doesn't take away from anyone else's experience all the while it makes the user experience better and more in line to what almost everyone does.

**Comment by TRtomasz** [comment]
@Zcc09 I would avoid such defeinitive statements about user experience as this would change something in the player UI and users had more than 5 years to get used to current state.

**Comment by WiretapStudios** [comment]
Not really, they asked to have it as an option, not change the permanent normal state. For me, it's the opposite, this is the only player of 4+ I use on desktop without that feature, so I'd like to have it.

**Comment by Zcc09** [comment]
I would suggest you read the first comment I sent. I talk about adding a toggle for users who prefer to have that option. 

I really rather not get into an argument on which is superior I just submitted this issue to show the devs that I'm interested in it and it's up to them to implement it or not.

**Comment by thisisrichardferreira** [comment]
@TRtomasz +1 from me for this one, any plans to implement a toggle for this? I often watch things on the couch with only a mouse and this would be very convenient, especially since play/pause from my headphones doesn’t work (macOS app). Out of all other players and streaming services I use, Streamio is the only one that doesn’t support clicking on the video to pause it.

**Comment by TRtomasz** [comment]
implemented in stremio 5

**Comment by thisisrichardferreira** [comment]
Thank you @TRtomasz! I struggled to find Streamio 5 for macOS but switched to web.streamio.com just for this feature, it's been working great.
